packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -159,6 +158,12 @@ export class Summarizer extends EventEmitter implements ISummarizer {
         this.stopDeferred.resolve(reason);
     }
 
+    public close() {
+        // This will result in dispose() call and "summarizerClientDisconnected" stop reason,
+        // unless stop() was called earlier
```

Comment:
Not sure what you mean by unless stop() was called earlier. Does it just mean that the reason will be something other than "summarizerClientDisconnected"?

Reply:
If stop() was not called up till this moment, then stop() call in dispose() will result in "summarizerClientDisconnected" reason being recorded in telemetry